src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -363,7 +363,10 @@ public static function create($uri, $method = 'GET', $parameters = array(), $coo
             case 'POST':
             case 'PUT':
             case 'DELETE':
-                if (!isset($server['CONTENT_TYPE'])) {
+                if (isset($server['content_type']) {
```

Comment:
I was thinking about it, but this seems the most conservative change.
Probably there is so many software using lowercase keys, that could break.